ydiff.py

```diff
@@ -28,9 +28,6 @@
                      'stdin, with side by side and auto pager support')
 }
 
-if sys.hexversion < 0x03000000:
```

Comment:
Please keep this check, build (package) dependencies and runtime dependencies are different. Others look good. Thank you.

Reply:
@ymattw `python_requires='>=3',` is runtime, are you sure you want me to drop that commit?

Reply:
I meant keeping the check in ydiff.py, which can be used without installing the pypi package.

Reply:
@ymattw I'm not sure that's worth protecting with all Pythons dead before 3.8, but can do. Pushed.